docs/.gitignore

```diff
@@ -1 +1,10 @@
-book
+.DS_Store
```

Comment:
copied from the [apollo-server docs gitignore](https://github.com/apollographql/apollo-server/blob/main/docs/.gitignore)

Reply:
i would generally remove `.DS_Store` from the .gitignore because it is operating system specific and should be in an individuals' global .gitignore but i won't let it block this